resources/lang/pl_PL/models.yml

```diff
@@ -25,6 +25,9 @@ facility:
   name: "$t(models.generic.name)"
   url: "adres url wewnątrz Memo"
   members: "osoby przypisane do placówki"
+  page: "strona placówki"
```

Comment:
Idea jest taka że grupa generic jest do rzeczy tak ogólnych że z dużym prawdopodobieństwem będą występować w wielu różnych modelach. Więc myślę że te trzy rzeczy raczej takie nie są i powinny być bezpośrednio w modelu do którego należą.

A swoją drogą model który ty wyświetlasz to nie jest members tylko facilities, i tam jest już trochę rzeczy, a jak czegoś brakuje to uzupełnij.

Reply:
Nie wiem do czego komentarz, bo ja tutaj edytuję 'facility' 🤔 (nie generic i nie members)

Grupę generic edytuję tylko wyżej modifiedAt na updatedAt, bo zarówno facilities jak i users tak mają w API, więc zakładam że taki będzie standard.